frontend/src/consts.ts

```diff
@@ -0,0 +1,4 @@
+// Team Event constants
```

Comment:
This was long overdue. But don't we already have a `constants.d.ts`/`constants.js`? What's up with those? Can we combine?

Maybe another ticket to investigate and factor out more constants.

Reply:
So that file is in `common-types` which is intended to hold shared types/data/info/etc across more than one project. We can definitely look into investigating constants that are shared across projects but in this case, this is a frontend only constant so I think leaving it here should be fine.